tamr_unify_client/models/project/step.py

```diff
@@ -0,0 +1,68 @@
+class ProjectStep:
+    """ A step of a Unify project.
+
+    See https://docs.tamr.com/reference#retrieve-downstream-dataset-usage
+
+    :param client: Delegate underlying API calls to this client.
+    :type client: :class:`~tamr_unify_client.Client`
+    :param data: The JSON body containing project step information.
+    :type data: :py:class:`dict`
+    """
+
+    def __init__(self, client, data):
+        self.client = client
+        self._data = data
+
+    @property
+    def project_step_id(self):
+        """:type: str"""
+        return self._data.get("projectStepId")
+
+    @property
+    def project_step_name(self):
+        """:type: str"""
+        return self._data.get("projectStepName")
+
+    @property
+    def project_name(self):
+        """:type: str"""
+        return self._data.get("projectName")
+
+    @property
+    def type(self):
+        """One of:
+            ``"SCHEMA_MAPPING"``
```

Comment:
This list will change out from under us, so we shouldn't embed it here.
Stated another way, Python Client doesn't control the contract for what the valid values are for this string. Since Python Client doesn't control it, Python Client can't actually assert what the valid values are. We should refer to the contract, rather than replicate it.

Reply:
This should be changed in the project file then too. I can do that during the refactor.